tests/test_commands.py

```diff
@@ -305,6 +305,18 @@ def test_asyncio_enabled_false(self):
         log = self.get_log(self.debug_log_spider, args=[])
         self.assertNotIn("Using reactor: twisted.internet.asyncioreactor.AsyncioSelectorReactor", log)
 
+    def test_custom_asyncio_loop_enabled_true(self):
+        log = self.get_log(self.debug_log_spider, args=[
+            '-s', 'TWISTED_REACTOR=twisted.internet.asyncioreactor.AsyncioSelectorReactor', '-s', 'ASYNCIO_EVENT_LOOP=uvloop.Loop'
+        ])
+        self.assertIn("Using asyncio event loop: uvloop.Loop", log)
+
+    def test_custom_asyncio_loop_enabled_false(self):
+        log = self.get_log(self.debug_log_spider, args=[
+            '-s', 'TWISTED_REACTOR=twisted.internet.asyncioreactor.AsyncioSelectorReactor'
+        ])
+        self.assertIn("Using asyncio event loop: asyncio.unix_events._UnixSelectorEventLoop", log)
```

Comment:
We need to make this check cross-platform.

Reply:
I found that asyncio uses different loops for unix and windows. Maybe its better to change the function and check for the absence of uvloop (which is not provided in the settings in this function) in the logs instead.

Reply:
I think a better approach would be to get the default loop class from asyncio programmatically, and check that the one reported one matches that one.

Reply:
Done